data/json/items/ammo/454.json

```diff
@@ -0,0 +1,25 @@
+[
```

Comment:
Just an idea at this point.
I was thinking more of a cache than a build to keep json loading dynamic,
so that we don't raise the barrier to entry for modders.